src/libData/DataStructures/TraceableDB.cpp

```diff
@@ -115,11 +128,13 @@ bool TraceableDB::RefreshDB() {
 
 void TraceableDB::DetachedExecutePurge() {
   LOG_MARKER();
+
   auto detached_func = [this]() -> void {
     if (!m_purgeRunning) {
+      unordered_set<dev::h256> inserted;
       m_stopSignal = false;
       m_purgeRunning = true;
-      ExecutePurge(0, true);
+      ExecutePurge(0, inserted, true);
```

Comment:
Hi @KaustubhShamshery , Just to confirm here,  `inserted` set is not of any use in `DetachedExecutePurge`. It is there to keep for the sake of the signature of function `ExecutePurge` and not to duplicate the code around. Right?